src/test/app/Escrow_test.cpp

```diff
@@ -671,18 +674,84 @@ struct Escrow_test : public beast::unit_test::suite
     }
 
     void
-    testMeta()
+    testMetaAndOwnership()
     {
-        testcase ("Metadata");
-
         using namespace jtx;
         using namespace std::chrono;
-        Env env(*this, with_features(featureEscrow));
 
-        env.fund(XRP(5000), "alice", "bob", "carol");
-        env(condpay("alice", "carol", XRP(1000), makeSlice(cb1), env.now() + 1s));
-        auto const m = env.meta();
-        BEAST_EXPECT((*m)[sfTransactionResult] == tesSUCCESS);
+        auto const alice = Account("alice");
+        auto const carol = Account("carol");
+
+        {
+            testcase ("Metadata & Ownership (without fix1523)");
+            Env env(*this, with_features(featureEscrow));
+            env.fund(XRP(5000), alice, carol);
+            auto const seq = env.seq(alice);
+
+            env(condpay(alice, carol, XRP(1000), makeSlice(cb1), env.now() + 1s));
+
+            auto const m = env.meta();
+            BEAST_EXPECT((*m)[sfTransactionResult] == tesSUCCESS);
+
+            auto const escrow = env.le(keylet::escrow(alice.id(), seq));
+            BEAST_EXPECT(escrow);
+
+            // Verify that Alice's owner directory tracks the escrow entry
+            ripple::Dir aod (*env.current(), keylet::ownerDir(alice.id()));
+            BEAST_EXPECT(std::distance(aod.begin(), aod.end()) == 1);
+            BEAST_EXPECT(std::find(aod.begin(), aod.end(), escrow) != aod.end());
+
+            // Verify that Carol doesn't have an owner directory at all.
+            ripple::Dir cod (*env.current(), keylet::ownerDir(carol.id()));
+            BEAST_EXPECT(cod.begin() == cod.end());
+        }
+
+        {
+            testcase ("Metadata (with fix1523, to self)");
+
+            Env env(*this, with_features(featureEscrow));
```

Comment:
Should this be `with_features(featureEscrow, fix1523)` to match the message on L710?

Reply:
Yes.